pkg/apis/garden/v1beta1/types.go

```diff
@@ -1637,6 +1637,9 @@ const (
 // KubeletConfig contains configuration settings for the kubelet.
 type KubeletConfig struct {
 	KubernetesConfig `json:",inline"`
+	// VolumePluginDir is the full path of the directory in which to search for additional third party volume plugins.
+	// +optional
+	VolumePluginDir *string `json:"volumePluginDir,omitempty"`
```

Comment:
Can you please also define it in `pkg/apis/core/v1alpha1/types_shoot.go`? 
Also, please run `./hack/generate-code` and update `example/90-shoot.yaml` accordingly.
Moreover, you have to inject the value given in the `Shoot` resource to the Helm chart [above](https://github.com/gardener/gardener/pull/1540/files#diff-7f7df7e11f6d06819b3292808517dd74R25) which is done here: https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/operatingsystemconfig.go#L277

Reply:
> Can you please also define it in `pkg/apis/core/v1alpha1/types_shoot.go`?

Done.

> Also, please run `./hack/generate-code` and update `example/90-shoot.yaml` accordingly.

There are some changes generated in `hack/api-reference` which seem unrelated. Should I add these as well? Otherwise done.

> Moreover, you have to inject the value given in the `Shoot` resource to the Helm chart

Done.